lib/react-native-multi-select.js

```diff
@@ -705,7 +707,7 @@ export default class MultiSelect extends Component {
                     </Text>
                     <Icon
                       name={hideSubmitButton ? 'menu-right' : 'menu-down'}
-                      style={styles.indicator}
+                      style={[styles.indicator, { right: -20, height: '100%' }]}
```

Comment:
I think this can be bad for other users of the library.
I think better to you create another `props`

Reply:
Yes, you are right but some devices has redundant indicator right margin.